### PR TITLE
Improve error message if ZenML repo is missing inside kubeflow container entrypoint

### PR DESCRIPTION
**Pull request by schustmi**
## Pre-requisites
Please ensure you have done the following:
- [x] I have read the **CONTRIBUTING.md** document.
- [x] If my change requires a change to the documentation, I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Describe changes
Improve error message if ZenML repo is missing inside kubeflow container entrypoint.

### CODE REVIEW

File: src/zenml/integrations/kubeflow/container_entrypoint.py [hunk 1]
```diff
@@ -475,7 +476,19 @@ def main() -> None:
...
+    try:
+        repo = Repository()
+    except RepositoryNotFoundError:
+        raise RepositoryNotFoundError(
+            "Unable to find ZenML repository in docker container. Make sure "
+            "your ZenML repository is not included in the dockerignore file "
+            "used when building the docker image to run this pipeline using "
+            "Kubeflow Pipelines. This dockerignore might be specified using "
+            "the `@pipeline` decorator or a file named '.dockerignore' at the "
+            "root of your local ZenML repository."
```
Comment (AlexejPenner) [inline src/zenml/integrations/kubeflow/container_entrypoint.py:488]: Not every user might know what the ZenML repository is - might be worth specifically saying .zen directory at some point in the message.


```suggestion
            "Unable to find ZenML repository in docker container. Make sure "
            "your ZenML repository (.zen directory) is not included in the dockerignore file "
            "used when building the docker image to run this pipeline using "
            "Kubeflow Pipelines. This dockerignore might be specified using "
            "the `@pipeline` decorator or a file named '.dockerignore' at the "
            "root of your local ZenML repository."
```

**Review by AlexejPenner — CHANGES_REQUESTED** [review]
You are fixing these faster than I can produce them :D - I would have a tiny suggestion to make this more beginner-friendly

File: src/zenml/integrations/kubeflow/container_entrypoint.py (hunk 1 above)
Comment (schustmi) [reply src/zenml/integrations/kubeflow/container_entrypoint.py:488]: Good idea, I'll fix it without the github suggestion to make sure the text doesn't overlap our 80 character line limit :)

**Review by htahir1 — APPROVED** [review]
Small but useful update!

**Pull request merged**